### ISSUE TITLE
Apps in chroot unable to create/find their own directories.

### ISSUE THREAD

**Issue by 62f** [ish-app/ish#839]
The workflow pasted below is only a single example of the general issue. Neither any actual app mentioned, nor entering an Ubuntu-in-ish environment will be necessary to replicate this very broad issue. There is a workaround: to manually create the empty directories if their names and locations can be guessed, though this always requires running the app twice and visually parsing the error, at which point the apps can finally locate and populate them with files. Note that this is not a case of being able to create a directory and losing track of it: there are no orphaned directories being created. $sudo elevation does not prevent this issue, and despite similitude to, this is not a repeat of issue #838 Some output adjusted for legibility.
From within a chroot:

```
root@iPhone:/alpine/root/ubuntu-in-ish/ubuntu-fs/home# cd au

root@iPhone:/alpine/root/ubuntu-in-ish/ubuntu-fs/home/au# dpkg -i au_0.1_all.deb

Selecting previously unselected package au.
(Reading database ... 4093 files and directories currently installed.)

Preparing to unpack au_0.1_all.deb ...
Unpacking au (0.1) ...

dpkg: error processing archive au_0.1_all.deb (--install):
 unable to create '/data/data/com.termux/files/usr/bin/au.dpkg-new' (while processing 'data/data/com.termux/files/usr/bin/au'): No such file or directory
Errors were encountered while processing:
 au_0.1_all.deb
```

**Comment by tbodt** [comment]
This appears to be because you tried to install a .deb intended for Termux on a regular Ubuntu chroot. I don't think that can possibly work.

**Comment by tbodt** [comment]
(leaving this open because I don't understand this perfectly and might be wrong)

**Comment by 62f** [comment]
Sadly this was only happening during a chroot session in Alpine 3.8, but since losing that setup and using the minifs to load Alpine 3.12, the file bug is now permanent with every one of the Alpine apps, and I now believe it’s likely to be unrelated to iSH (73). As a test I will downgrade iSH as far as possible to see if the bug will fix itself, but 3.12 definitely brought some other  productivity-killing quirks along that I’m certain will neither simply evaporate.

**Comment by tbodt** [comment]
Let me know if you have specific repro steps. The error in the original post is something I would expect from attempting to install a termux package on ubuntu.
